rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -245,23 +248,17 @@ def inform_user_missing_acquisition(
             self,
             asset: Asset,
             time: Timestamp,
-            found_amount: Optional[FVal] = None,
-            missing_amount: Optional[FVal] = None,
+            found_amount: FVal,
+            missing_amount: FVal,
     ) -> None:
-        """Inform the user for missing data for an acquisition via the msg aggregator"""
-        if found_amount is None:
-            self.msg_aggregator.add_error(
-                f'No documented acquisition found for {asset} before '
-                f'{self.timestamp_to_date(time)}. Let rotki '
-                f'know how you acquired it via a ledger action',
-            )
-            return
-
-        self.msg_aggregator.add_error(
-            f'Not enough documented acquisitions found for {asset} before '
-            f'{self.timestamp_to_date(time)}. Only found acquisitions '
-            f'for {found_amount} {asset} and miss {missing_amount} {asset}.'
-            f'Let rotki know how you acquired it via a ledger action',
+        """Inform the user for missing data for an acquisition by appending it to a list."""
+        self.missing_acquisitions.append(
```

Comment:
As I said in discord, since this function is now just appending to a list, remove the function completely and use the list appending directly. No need for the extra work.